scripts/fetch/get_dv_data.R

```diff
@@ -0,0 +1,12 @@
+#' Get NM daily data
+#'
+#'@param viz the vizlab object 
+fetcher.get_dv_data <- function(viz = getContentInfo(viz.id = "get_NM_data")){
```

Comment:
wait! One other thing - in water-use, these functions are called with `fetch` not `fetcher`. @jiwalker-usgs which is correct? I know I've seen it both ways